cargo-espflash/src/main.rs

```diff
@@ -88,7 +88,7 @@ enum Commands {
     Flash(FlashArgs),
     /// Hold the target device in reset
     HoldInReset(ConnectArgs),
-    /// List serial ports available for flashing.
+    /// List serial ports available.
```

Comment:
```suggestion
    /// List available serial ports.
```

If we're changing this anyway, this sounds more natural to me